parts/chapters/subsections/6.3/MULTREGD.fodt

```diff
@@ -4399,10 +4399,13 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
          <text:p text:style-name="P19104">Where REGION1 and REGION2 should be less than or equal to the maximum number of regions as defined on the REGDIMS keyword for the FIPNUM and OPERNUM arrays or the GRIDOPTS keyword for the MULTNUM array in the RUNSPEC section.</text:p>
         </text:list-item>
         <text:list-item>
-         <text:p text:style-name="P19104">If REGION1 is defaulted then the transmissibilities between REGION2 and all other regions will be multiplied. Similarly, if REGION2 is defaulted then the transmissibilities between REGION1 and all other regions will be multiplied. If both REGION1 and REGION2 are defaulted then the transmissibilities between all pairs of regions will be multiplied.</text:p>
+         <text:p text:style-name="P19104">If REGION1 is defaulted then the diffusivities between REGION2 and all other regions will be multiplied. Similarly, if REGION2 is defaulted then the diffusivities between REGION1 and all other regions will be multiplied. If both REGION1 and REGION2 are defaulted then the diffusivities between all pairs of regions will be multiplied.</text:p>
         </text:list-item>
         <text:list-item>
-         <text:p text:style-name="P19104">If REGION1 and REGION2 are equal and positive then all the transmissibilities within that region as well as the transmissibilities between that region and all other regions will be multiplied.</text:p>
+         <text:p text:style-name="P19104">If REGION1 and REGION2 are equal and positive then all the diffusivities within that region as well as the diffusivities between that region and all other regions will be multiplied.</text:p>
+        </text:list-item>
+        <text:list-item>
+         <text:p text:style-name="P19104">If the same combination of REGION1 and REGION2 is repeated then only the latest multiplier will be used.</text:p>
```

Comment:
How about this?

Reply:
That was not what I meant. I meant that for this
```
MULTREGD                                                                    
      1       2       1.05    1*     'ALL'   M           /                                                                
      1       2       1.05    1*     'ALL'   F           /
/
even thought there is an F instead of Min the second directive, the last instruction wins.

Reply:
Sorry, got the wrong end of the stick